lib/transport.rb

```diff
@@ -0,0 +1,37 @@
+class Transport
+
+  include Comparable
+
+  attr_accessor :current_weight , :available
+
+  CONVERT_TO_MINUTES = 60 
+
+  def initialize(current_weight , is_available)
+    @current_weight = current_weight
+    @available = is_available
+  end
+  
+  def max_weight
+    raise NotImplementedError
+  end
+
+  def max_weight=(value)
+    raise NotImplementedError
+  end
+
+  def speed
+    raise NotImplementedError
+  end
+
+  def speed=(value)
+    raise NotImplementedError
+  end
+
+  def delivery_time(distance)    
+    return (distance / @speed) * CONVERT_TO_MINUTES 
+  end
+
+  def <=> (other)
```

Comment:
It should work in other way due to specification
```
расширить класс Transport чтобы объекты этого класса можно было сравнивать по скорости доставки 1 кг посылки
```

Reply:
rewrote this method